src/libs/Navigation/CustomActions.js

```diff
@@ -57,6 +59,11 @@ function getScreenNameFromState(state) {
     return getRouteFromState(state).name || '';
 }
 
+let isSmallScreenWidth;
+Dimensions.addEventListener('change', ({window}) => {
+    isSmallScreenWidth = window.width <= variables.mobileResponsiveWidthBreakpoint;
+});
+
```

Comment:
@Santhosh-Sellavel @aldo-expensify I adding a listener to get the screen size. It's should reflect the screen size.

Reply:
hum, I still don't understand how `Dimensions.get('window');` would be outdated. From the documentation it looks like it should return the current width, up to date when is used by `pushDrawerRoute`.

I don't think this is necessary at all.

@Santhosh-Sellavel can you explain more why you think it would "not reflect correctly"?

Reply:
So let's go with the first way right @aldo-expensify or let's keep this? I'll leave it to you to decide.

Reply:
Cool! @mollfpr kindly update the PR